internal/api/migrate/testdata/tables_perses_dashboard.json

```diff
@@ -134,6 +167,7 @@
                   "name": "bonjour"
                 }
               ],
+              "cellSettings": [],
```

Comment:
to avoid generating such empty arrays you could use an intermediary var in the migration file

aka replace the current

```cue
cellSettings: list.FlattenN([...], 1)
```
by
```cue
let x = list.FlattenN([...], 1)
if len(x) > 0 {
    cellSettings: x
}
```